test/WormholeL1ERC20Bridge.t.sol

```diff
@@ -44,6 +42,18 @@ contract L1ERC20BridgeTest is Constants, WormholeRelayerBasicTest {
   FakeERC20 l1Erc20;
   WormholeL1ERC20Bridge l1Erc20Bridge;
 
+  event Deposit(
+    address indexed sender,
+    address indexed targetAddress,
+    uint256 indexed targetChain,
+    uint256 amount,
+    uint256 deliveryCost
+  );
+
+  event WormholeMessageReceived(
+    uint16 indexed sourceChain, bytes32 indexed sourceAddress, bytes32 deliveryHash
+  );
```

Comment:
Is this event being tested?

Reply:
Ah actually, I think this should be removed right?

Reply:
removed, thank you!